TribalWars/src/Infrastructure/ViewModels/BuildingRequiredBuildingViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using ApplicationCore.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.ViewModels
```

Comment:
Classes like ViewModels belong to domain project, so should be located in ApplicationCore.